TestControllers/TestController.psm1

```diff
@@ -1087,6 +1093,6 @@ Class TestController {
 			Write-LogErr "Source: Line $line in script $script_name."
 		}
 
-		return $retValue
+		return @($retValue,$deltaLogs)
```

Comment:
it is always an empty array?

Reply:
for the public platforms, yes. We can also change it to return call trace, or return error/failure/warning messages. Do you have suggestions?